src/components/Pagination/utils.tsx

```diff
@@ -0,0 +1,11 @@
+import { PaginationOption, PaginationOptionValue } from './Pagination';
+
+export const determineOptionValue = (option: PaginationOption | PaginationOptionValue) => {
```

Comment:
would we ever want to export this from seldon? I guess not, since `onChange` is just getting passed the value...